etcd/tests/conftest.py

```diff
@@ -28,12 +28,17 @@ def dd_environment(instance):
         yield instance
 
 
+@pytest.fixture(scope='session')
+def legacy_instance():
+    return {'url': URL, 'use_preview': False}
+
+
 @pytest.fixture(scope='session')
 def instance():
     if V3_PREVIEW:
         return {'use_preview': True, 'prometheus_url': '{}/metrics'.format(URL)}
     else:
-        return {'url': URL}
+        return legacy_instance
```

Comment:
`TypeError: <function legacy_instance at 0x101fc56e0> is not JSON serializable`

Try to pass `legacy_instance` in the function parameters

Reply:
Nah, what I meant to do here is legacy_instance()

Reply:
Yeah both work in that case, because `legacy_instance` is a session-scoped fixture

Reply:
Which you cannot, do. I'll extract a constant then -.-